rbac/management/role/serializer.py

```diff
@@ -112,7 +112,8 @@ def _should_add_hierarchy(self, instance):
 
     def _is_workspace_filter(self, instance):
         is_workspace_application = instance.application == settings.WORKSPACE_APPLICATION_NAME
-        is_workspace_resource_type = instance.resource_type == settings.WORKSPACE_RESOURCE_TYPE
+        # Accept both 'groups' and '*' as valid resource types for inventory workspace hierarchy
+        is_workspace_resource_type = instance.resource_type in [settings.WORKSPACE_RESOURCE_TYPE, "*"]
```

Comment:
@astrozzc Is it not better to convert `WORKSPACE_RESOURCE_TYPE` into a list? So that if a new resource type is created and allowed to deal with workspaces, the code will be ready.

Reply:
Good point! Updated!